src/main/java/javaClasses/homework_4/Developer.java

```diff
@@ -0,0 +1,19 @@
+package javaClasses.homework_4;
+
+public class Developer extends Staff {
+    private  String techstack;
+
+    protected String getTechstack() {
```

Comment:
In this case it's ok to use protected access modifier, but usually you call methods from different packages, so better choice would public modifier

Reply:
done